run_pipeline.sh

```diff
@@ -3,6 +3,7 @@
 _S3_FILE_URI="$1"
 _FILE_ID="$2"
```

Comment:
What is _FILE_ID for? I don't remember anything about that parameter. As proof, there is nothing about it in runner.py.